package.json

```diff
@@ -19,6 +19,7 @@
     "express": "^4.16.3",
     "history": "^4.6.1",
     "husky": "^0.14.3",
+    "image-size": "^0.6.2",
     "jwt-decode": "^2.2.0",
```

Comment:
@erathinam The `image-size` package shouldn't need to be in here - especially now that you've fixed the `lint-staged` NPM task.